respondd.py

```diff
@@ -78,6 +79,9 @@ def handle(self):
     parser.add_argument('-m', dest='mesh_ipv4',
                         metavar='<mesh_ipv4>',
                         help='mesh ipv4 address')
+    parser.add_argument('-n', dest='domain_code',
+                        metavar='domain_code',
+                        help='Domain Code for system/domain_code')
```

Comment:
There should be a reasonable default for this value. I'd suggest setting it to `'default'` when `-n` is not specified.

Reply:
Do we need a default? If nothing is set, there should be no domain transmitted (as it is now)?

Reply:
Hm, ok. That is a side effect of having a `None` value in there then. I would have expected it to break due to an exception somewhere in handler processing when a handler returns `None`.

I'll add an `is_enabled` method to data sources so they can signal wether they should be called or not. This seems a lot clearer to me than handling `None` values all the time.